### PR TITLE
Add Chinese Generation Task

### PR DESCRIPTION
**Pull request by YY0649**

### CODE REVIEW

File: src/tasks/__init__.py [hunk 1]
```diff
@@ -34,7 +34,13 @@
     "flare_zh_corpus": flare.ZHBQcourse,
     "flare_zh_fineval": flare.ZHFinEval,
     "flare_zh_afqmc": flare.ZHAFQMC,
-    "flare_zh_stockb": flare.ZHstock11,
```
Comment (jiminHuang) [inline src/tasks/__init__.py:37]: Why stockb is removed?

File: src/tasks/flare.py [hunk 2]
```diff
@@ -1008,3 +1009,109 @@ class ZHFinEval(Classification):
...
+            all_preds.extend(pred)
+            
+        return set(all_golds), set(all_preds)
+
+    def precision(self, items):
+        golds, preds = self.process(items)
+        gold_len = len(golds)
+        count = 0
+
+        for gold_item in golds:
```
Comment (jiminHuang) [inline src/tasks/flare.py:1059]: is it intersection? You can directly use & to calculate

File: src/tasks/zhutils.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    # Initialize
+    name = entity_string.split(',')[0]
+    if len(entity_string.split(',')) > 1 and entity_string.split(',')[1]:
+        entity_type = entity_string.split(',')[1].strip()
+    else:
+        entity_type = 0
+    formatted_name = ' '.join(list(name))
+    formatted_result = f"{formatted_name}, {entity_type}"
+
+    entity_list = [(", ".join(val.split(", ")[:-1]), val.split(", ")[-1]) for val in formatted_result.split("\n")]
```
Comment (jiminHuang) [inline src/tasks/zhutils.py:11]: Should we consider the Chinese ，

**Review by jiminHuang — CHANGES_REQUESTED** [review]

File: src/tasks/flare.py [hunk 4]
```diff
@@ -1008,3 +1009,109 @@ class ZHFinEval(Classification):
...
+        return prec
+
+    def recall(self, items):
+        golds, preds = self.process(items)
+        pred_len = len(preds)
+        count = 0
+
+        for gold_item in golds:
+            for pred_item in preds:
+                if gold_item & pred_item:
```
Comment (jiminHuang) [inline src/tasks/flare.py:1075]: The `process` returns two sets?

**Review by xashely — APPROVED** [review]

**Review by jiminHuang — APPROVED** [review]

**Pull request merged**